src/kernels/conv3x3.s

```diff
@@ -58,29 +58,10 @@ gid_z = 4
 // dwords 0:1 - input buffer pointer
 // dwords 2:3 - weights pointer
 // dwords 4:5 - output buffer pointer
-// dwords 6:7 - debug buffer pointer
 .set in_ptr_off, 0x0
 .set wei_ptr_off, 0x8
 .set out_ptr_off, 0x10
 
-.ifnotdef no_params_file
```

Comment:
`no_params_file` must be removed from the solver.

Reply:
Thanks for the quality review, fixed in c41f359